content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -124,38 +130,36 @@ into the plots file `evaluation/plots/prc.json`:
 
 Similarly, it writes arrays for the
 [roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
-into `evaluation/plots/sklearn/roc.json`,
+and
 [confusion matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
-into `evaluation/plots/sklearn/confusion_matrix.json`, and an image
-`evaluation/plots/importance.png` with a feature importance bar chart for
-additional plots.
-
-> DVC doesn't force you to use any specific file names, nor does it enforce a
-> format or structure of a metrics or plots file. It's completely
-> user/case-defined. Refer to `dvc metrics` and `dvc plots` for more details.
-
-You can view tracked metrics and plots with DVC. Let's start with the metrics:
+into JSON files in the `eval/live/plots` directory, and an image
+`eval/importance.png` with a feature importance bar chart for additional plots.
```

Comment:
💅🏼 and a `eval/importance.png` image showing...

Reply:
> for additional plots

What does that mean? I.e. are they saved to another plots file? Which one?

Reply:
p.s. More in general, if we could combine this paragraph with the previous one (before the code block), the section would be more readable: just a p and a code block before we get to the main point (configuring plots), next 👇🏼

Reply:
Specifying all the plots files in one paragraph is a good idea. Do you want to add that to #4207?